src/fb5.js

```diff
@@ -68,6 +69,14 @@ function setFeedObserver() {
 }
 
 function onPageChange() {
+  if (isFBWatch()) {
+    onPageChangeInWatch();
```

Comment:
If a user is navigating to `/watch`, then we should clean up `feedObserver`.

I'm not sure how you want to refactor this, we could run `onPageChangeInNewFeed` one more time so it will detect that the news feed object is not there and run `feedObserver.disconnect()`.

Or you could refactor cleaning up into a new function, and call it from the old place and here.

Reply:
ok will check